src/main/scala/mesosphere/marathon/core/DefaultCoreModule.scala

```diff
@@ -0,0 +1,94 @@
+package mesosphere.marathon.core
+
+import akka.actor.ActorSystem
+import com.google.inject.Inject
+import mesosphere.marathon.core.task.tracker.KillOverdueStagedTasksActor
+import mesosphere.marathon.metrics.Metrics
+import mesosphere.marathon.{ MarathonConf, MarathonSchedulerDriverHolder }
+import mesosphere.marathon.api.LeaderInfo
+import mesosphere.marathon.core.base.actors.ActorsModule
+import mesosphere.marathon.core.base.{ Clock, ShutdownHooks }
+import mesosphere.marathon.core.flow.FlowActors
+import mesosphere.marathon.core.launcher.LauncherModule
+import mesosphere.marathon.core.launchqueue.LaunchQueueModule
+import mesosphere.marathon.core.leadership.LeadershipModule
+import mesosphere.marathon.core.matcher.manager.OfferMatcherManagerModule
+import mesosphere.marathon.core.task.bus.TaskBusModule
+import mesosphere.marathon.state.AppRepository
+import mesosphere.marathon.tasks.{ TaskFactory, TaskTracker }
+
+import scala.util.Random
+
+/**
+  * Provides the wiring for the core module.
+  *
+  * Its parameters represent guice wired dependencies.
+  * [[CoreGuiceModule]] exports some dependencies back to guice.
+  */
+class DefaultCoreModule @Inject() (
```

Comment:
I don't like the name "Default" here. But since I don't have a better proposal...